src/nvim/eval.lua

```diff
@@ -1047,7 +1047,7 @@ M.funcs = {
 
       Can also be used as a |method|: >
       	GetName()->byteidxcomp(idx)
-      <
```

Comment:
I'd prefer to keep these, for better treesitter parseability.

Reply:
What "keep"?

Reply:
Like I said, we can fix all this in follow ups. For this PR the aim is to have a minimal diff.

Reply:
Well, quite a few were added, so consistency is good (if you add them anyway). My point is since they _were_ already added, explicitly removing them again just for the sake of minimizing the diff is a net loss.

Reply:
🤷 . They will all get fixed anyway.